src/main/java/seedu/address/model/student/LessonTime.java

```diff
@@ -104,6 +105,24 @@ public boolean equals(Object other) {
                 && lessonEnd.equals(otherLessonTime.getLessonEnd());
     }
 
+    @Override
+    public int compareTo(LessonTime o) {
+
```

Comment:
Maybe can add a check for whether o is the same as this and return 0 immediately if so?